src/app/beer_garden/garden.py

```diff
@@ -33,7 +36,34 @@ def get_gardens() -> List[Garden]:
         The Garden list
 
     """
-    return db.query(Garden)
+
+    query_results = db.query(Garden)
+    if query_results:
+        query_results.append(get_local_garden())
+        return query_results
+    return [get_local_garden()]
+
+
+def get_local_garden() -> Garden:
+    """Retrieved the local garden object
+
+    Returns:
+        Garden Object
+
+    """
+    local_garden = Garden(
+        name=config.get("garden.name"),
+        connection_type="LOCAL",
+        systems=get_systems(filter_params={"local": True}),
+        status="RUNNING",
+        status_info="This is your local Garden record, not persisted in the Database. "
+        "Changes will not be reflected",
```

Comment:
I think we should take this out - this field isn't actually a string. This feels like something we should just display in the frontend when see the `LOCAL` connection type.

Reply:
We will drop this field completely. I will see about throwing an alert when they go to edit a local instance.